server/metrics/metrics.go

```diff
@@ -27,6 +27,7 @@ type Metrics interface {
 	ObserveFilesCount(action, source, isDirect, discardedReason string, count int)
 	ObserveSyntheticUsersTotal(count int64)
 	ObserveLinkedChannelsTotal(count int64)
+	ObserveUpstreamUsersTotal(count int64)
```

Comment:
Why we have chosen Total and not count here ?

Reply:
`ObserveUpstreamUsersTotal` is a metrics whereas `ObserveFilesCount` is a counter metrics

Reply:
Add a same comment below these interface functions as well and segregate them as well. @ayusht2810